### ISSUE TITLE
Portal Necessity?

### ISSUE THREAD

**Issue by ChasonMindImp** [maltyeva/mindimp-pg#106]
@MindImpMatt and I were chatting about this just now. We have yet to use the student portal for anything, since logging in as an admin gets us all the access we need. We are wondering if it is worth keeping this function. It seems like an unnecessary extra button to get to what we need to do. Removing it may streamline the overall website experience for admins/instructors.

**Comment by maltyeva** [comment]
That's a really good point. The reason I have it is for ease of programming, so i know what the students see without logging into my other account, but I agree that it's not optimal. I do still think we need to do some things for a page redesign to make it more user-friendly and I'd love to hear ideas!

**Comment by ChasonMindImp** [comment]
I feel like this will have two layers: cosmetic stuff and workflow. The
former we may be able to get when the Chinese website is done, so they have
a similar visual vibe to them. The latter, my gut tells me, should be a
bigger meeting, rather than an amalgam of github comments. Would that be
more helpful overall? Or maybe have each of us play and put together a
"user report" and see if we use the sites similarly?

When we did rebrands and website designs in my previous position, one was
crowd-sourced (which was visual and functional hell) and the other was
dominated by the marketing department (which drew the ire of the academic
program side when they were told that workflow would be streamlined and
processes/visuals would be consistent across all pages). I think we have
been much more effective at the crowd-sourcing, but I feel like a happy
medium between the two sides will be best.

On Mon, Apr 27, 2015 at 5:02 PM, maltyeva <notifications@github.com> wrote:

> That's a really good point. The reason I have it is for ease of
> programming, so i know what the students see without logging into my other
> account, but I agree that it's not optimal. I do still think we need to do
> some things for a page redesign to make it more user-friendly and I'd love
> to hear ideas!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/106#issuecomment-96574461>.
>



-- 
Chason Dailey | 戴奇森
Program Manager and Advisor* | **思脉 **MindImp*
https://www.mindimp.com/

**Comment by maltyeva** [comment]
Yes, totally agree. I would definitely like to hear back in terms of user flow.
